recipe/install-libxgboost.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+. activate "${BUILD_PREFIX}"
```

Comment:
Do you know what is going on here? This was in the upstream recipe too IIRC, but IDK why.

Reply:
Seems like some paths are missing during copy but weirdly the CI build is behaving differently to my local installation, so I first need to find out what's the difference.

Reply:
any update?